### ISSUE TITLE
[Issue]: RHEL8 - Build/Ctest - Find OpenMP failure

### ISSUE THREAD

**Issue by kiritigowda** [ROCm/rpp#449]
### Problem Description

```
1: -- Found OpenMP_C: -fopenmp (found version "4.5")
1: CMake Error at /usr/local/lib64/python3.6/site-packages/cmake/data/share/cmake-3.25/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
1:   Could NOT find OpenMP_CXX (missing: OpenMP_CXX_FLAGS OpenMP_CXX_LIB_NAMES)
1: Call Stack (most recent call first):
1:   /usr/local/lib64/python3.6/site-packages/cmake/data/share/cmake-3.25/Modules/FindPackageHandleStandardArgs.cmake:600 (_FPHSA_FAILURE_MESSAGE)
1:   /usr/local/lib64/python3.6/site-packages/cmake/data/share/cmake-3.25/Modules/FindOpenMP.cmake:580 (find_package_handle_standard_args)
1:   CMakeLists.txt:65 (find_package)
```

## FIX

The fix is to install OpenMP before running the test on RHEL 8
yum install libomp libomp-devel
 
 
bash-4.4# ctest
Test project /tmp/grpc/rpp-test
    Start 1: rpp_qa_tests_tensor_host_all
1/2 Test #1: rpp_qa_tests_tensor_host_all .....   Passed   30.83 sec
    Start 2: rpp_qa_tests_tensor_hip_all
2/2 Test #2: rpp_qa_tests_tensor_hip_all ......   Passed  123.16 sec
 
100% tests passed, 0 tests failed out of 2
 

### Operating System

RHEL 8

### CPU

ALL

### GPU

AMD Instinct MI300

### ROCm Version

ROCm 6.0.0

### ROCm Component

rpp

**Comment by kiritigowda** [comment]
@r-abishek @LakshmiKumar23 -- should RPP packages bring in OpenMP packages?

**Comment by kiritigowda** [comment]
@r-abishek -- pre-reqs do not list OpenMP - https://github.com/ROCm/rpp/tree/develop/utilities/test_suite#prerequisites

**Comment by kiritigowda** [comment]
@r-abishek - let me know when this is implemented with #453

**Comment by r-abishek** [comment]
@kiritigowda Addressed by #453
